test/game/game_setup_test.exs

```diff
@@ -96,5 +96,18 @@ defmodule GameSetupTest do
       assert players ==
         %Players{player_one: %Player{type: HardComputerPlayer, symbol: "X"}, player_two: %Player{type: HardComputerPlayer, symbol: "O"}}
     end
+
+    test "displays invalid setup validation message via invalid input for player selection, then returns players containing selected players and associated symbols" do
```

Comment:
Can this be translated into the format of:
```ex
describe "Player selection" do
  test "shows an error message when a user enter an invalid input" do
  ...
  end
end
```
I suspect this pattern can also be applied to some other tests as well. Not suggesting that we update all of them just food for thought.

Reply:
Would these describes both be under the overall `game_setup/1` describe? Also, do you think this is a good refactor indication where player selection and difficulty selection should be their own modules which game setup can use? I hope this all makes sense, thank you for all the help! 😄

Reply:
I think Im gonna make this refactor its own PR if thats ok, I want to try and break player and difficulty selection into their own separate modules and I think that would be easier to follow if it was in another PR! What are your thoughts? 😄